homeassistant/components/tado/__init__.py

```diff
@@ -121,6 +123,29 @@ async def async_unload_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     return unload_ok
 
 
+async def async_migrate_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> bool:
+    """Migrate old entry."""
+    _LOGGER.debug("Migrating from version %s for %s", config_entry.version, DOMAIN)
+
+    if config_entry.version == 1:
```

Comment:
We should avoid migrations if we can, and it looks like we can migrate this while fetching the data from the options.

Config entry migrations will prevent the user from rolling back Home Assistant versions so should be avoided unless really necessary.

Reply:
Sorry for the delay, not had any spare time.

Ahh ok, yeah that makes sense, I thought I was doing the right thing with migration flow, but yeah no worries, I'll take it out